hack/release.toml

```
@@ -155,6 +155,12 @@ machine:
     servers:
       - /dev/ptp0
 ```
+"""
+
+    [notes.ca-rotation]
+        title = "CA Rotation"
+        description = """\
+Talos Linux now supports rotating the root CA certificate and key for Talos API and Kubernetes API.
```

```suggestion
Talosctl now supports rotating the root CA certificate and key for Talos API and Kubernetes API.
```

since it's a client op :thinking: ?

---

well, Talos supports it, and talosctl facilitates that. You could rotate manually as well (I will add it to the docs in beta phase). 

And more than that, someone could attach their own CA, so they could issue `talosconfig`s from the PKI infra of their choice.